std_msgs/Header header 
float64 heading
geometry_msgs/Vector3 accel_bias
geometry_msgs/Vector3 gyro_bias